common/djangoapps/third_party_auth/models.py

```diff
@@ -882,10 +882,33 @@ class Meta(object):
 
 class ProviderApiPermissions(models.Model):
     """
-    This model links OAuth2 client with provider Id.
+    DEPRECATED: This model links OAuth2 client with provider Id.
 
     It gives permission for a OAuth2 client to access the information under certain IdPs.
 
+    This model is no longer in use.
+
+    TODO: BOM-1270: This model and table will be removed post-Juniper
+    so Open edX instances can migrate data if necessary.
+    To migrate, use ApplicationAccess and add a ``filter`` of the form
```

Comment:
Can you move this sentence as part of the steps below? Also, maybe adds some bullets to the steps below?